cmake/std/atdm/README.md

```diff
@@ -635,110 +633,14 @@ example, skip the configure, skip the build, skip running tests, etc.
 
 ## Specific instructions for each system
 
-* <a href="#ridewhite">ride/white</a>
-* <a href="#shillerhansen">shiller/hansen</a>
 * <a href="#tlcc-2-and-cts-1">TLCC-2 and CTS-1</a>
-* <a href="#mutrino">mutrino</a>
-* <a href="#sems-rhel6-environment">SEMS RHEL6 Environment</a>
 * <a href="#sems-rhel7-environment">SEMS RHEL7 Environment</a>
-* <a href="#spack-rhel-environment">Spack RHEL Environment</a>
 * <a href="#cee-rhel6-and-rhel7-environment">CEE RHEL6 and RHEL7 Environment</a>
 * <a href="#ats-2">ATS-2</a>
 * <a href="#astra-vanguard-arm-system">ASTRA (Vanguard ARM System)</a>
 * <a href="#ats-1">ATS-1</a>
 
 
-### ride/white
```

Comment:
Why is this being removed?  As long as the Trilinos PR tester is running on 'ride', this configuration should stay as it is currently running and posting to CDash.

Please don't touch anything that impacts the ATDM Trilinos builds posting to CDash until the Framework team has taken over full responsibility for these builds.

Reply:
When speaking with EMPIRE, GEMMA, and SPARC, they all indicated they have no need of the ride environments/configurations.  If you prefer, though, we can add the ride directories back for the time being.